mesonbuild/interpreter.py

```diff
@@ -1180,53 +1180,53 @@ def __init__(self, build, backend, subproject='', subdir='', subproject_dir='sub
+                           'error': self.func_error,
+                           'executable': self.func_executable,
+                           'dependency': self.func_dependency,
+                           'static_library': self.func_static_lib,
+                           'shared_library': self.func_shared_lib,
+                           'shared_module': self.func_shared_module,
+                           'library': self.func_library,
+                           'jar': self.func_jar,
+                           'build_target': self.func_build_target,
+                           'custom_target': self.func_custom_target,
+                           'run_target': self.func_run_target,
+                           'generator': self.func_generator,
+                           'test': self.func_test,
+                           'benchmark': self.func_benchmark,
+                           'install_headers': self.func_install_headers,
+                           'install_man': self.func_install_man,
+                           'subdir': self.func_subdir,
+                           'install_data': self.func_install_data,
+                           'install_subdir': self.func_install_subdir,
+                           'configure_file': self.func_configure_file,
+                           'include_directories': self.func_include_directories,
+                           'add_global_arguments': self.func_add_global_arguments,
+                           'add_project_arguments': self.func_add_project_arguments,
+                           'add_global_link_arguments': self.func_add_global_link_arguments,
+                           'add_project_link_arguments': self.func_add_project_link_arguments,
+                           'add_languages': self.func_add_languages,
+                           'find_program': self.func_find_program,
+                           'find_library': self.func_find_library,
+                           'configuration_data': self.func_configuration_data,
+                           'run_command': self.func_run_command,
+                           'gettext': self.func_gettext,
+                           'option': self.func_option,
+                           'get_option': self.func_get_option,
+                           'subproject': self.func_subproject,
+                           'vcs_tag': self.func_vcs_tag,
+                           'set_variable': self.func_set_variable,
+                           'is_variable': self.func_is_variable,
+                           'get_variable': self.func_get_variable,
+                           'import': self.func_import,
+                           'files': self.func_files,
+                           'declare_dependency': self.func_declare_dependency,
+                           'assert': self.func_assert,
+                           'environment': self.func_environment,
+                           'join_paths': self.func_join_paths,
+                          })
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->